src/pages/pricing.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="/assets/favicon.ico" type="image/x-icon" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,700&display=swap" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Permanent+Marker&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/public/css/styles.css">
+    <title>Pricing</title>
+</head>
+
+<body>
+    <div id="common-header"></div>
+    <section class="Banner">
+        <div class="Banner-sides Banner-first">
+        </div>
+        <div class="Banner-sides Banner-second">
+        </div>
+    </section>
+    <div class="Banner-food">
+        <h1 class="Banner-food-text">With Great Power Comes Great Responsibility</h1>
+    </div>
+    <section class="Pricing-plans">
+        <div class="Pricing-container">
+            <div class="Pricing-title">
+                <h2>Pricing Plans</h2>
+            </div>
+            <div class="Pricing-box">
+
+                <div class="Pricing-card">
+                    <h3 class="Pricing-card-title">Silver Package</h3>
```

Comment:
Use anothe tag instead of **h3** or **h2**, you can use span and then apply the style that you need

Reply:
done